src/hotspot/share/gc/shared/vmStructs_gc.hpp

```diff
@@ -28,7 +28,6 @@
 #include "gc/shared/ageTable.hpp"
 #include "gc/shared/cardTable.hpp"
 #include "gc/shared/collectedHeap.hpp"
-#include "gc/shared/genCollectedHeap.hpp"
```

Comment:
vmstructs purpose is to support SA. Thus renaming should be done here instead of deletion. You need to restore this line and rename genCollectedHeap.hpp -> serialHeap.hpp.

Reply:
It looks like the include of serialHeap.hpp is already taken care of by vmStructs_serial.hpp, so deleting this line looks like it is ok.